GCM/App/userInfo/UserInfo.java

```diff
@@ -1,7 +1,11 @@
 package userInfo;
 
+import users.User;
+
 public interface UserInfo {
 
 	boolean login(String username, String password);
```

Comment:
```suggestion
	/**
	* Executes a log-in request. If successful, it will update the UserInfo object with the logged in user's details
	* @return true if the log-in was successful, false otherwise
	*/
	boolean login(String username, String password);
```